precompiles/avs/IAVSManager.sol

```diff
@@ -169,14 +227,24 @@ interface IAVSManager {
         string memory operatorAddr
     ) external view returns (uint256 amount);
 
-    event TaskCreated(
-        uint64 taskId,
-        string taskContractAddress,
-        string name,
-        bytes hash,
-        uint64 taskResponsePeriod,
-        uint64 taskChallengePeriod,
-        uint64 thresholdPercentage,
-        uint64 taskStatisticalPeriod
-    );
+    /// @dev getAVSInfo  is a function to query Avs info.
+    /// @param avsAddr The address of the avs
+    function getAVSInfo(
+        address avsAddr
+    ) external view returns (string memory epochIdentifier);
+
+    /// @dev getTaskInfo  is a function to query task info.
+    /// @param taskAddr The address of the avs task
+    /// @param taskID The id of task.
+    function getTaskInfo(
+        address taskAddr,
+        uint64 taskID
```

Comment:
Isn't it enough using just the `taskAddr`?

Reply:
The task primary key is a unique combination of address and ID